pkg/cmd/infra/deployer/deployer_test.go

```diff
@@ -41,9 +42,10 @@ func TestDeployer_getDeploymentFail(t *testing.T) {
 }
 
 func TestDeployer_deployScenarios(t *testing.T) {
-	mkd := func(version int64, status appsapi.DeploymentStatus, replicas int32, desired int32) *kapi.ReplicationController {
+	mkd := func(version int64, status appsapi.DeploymentStatus, replicas int32, desired int32) *corev1.ReplicationController {
 		deployment := mkdeployment(version, status)
-		deployment.Spec.Replicas = int32(replicas)
+		replicasInt := int32(replicas)
```

Comment:
why do you need this? argument is of the same type, variable and local to a function